src/Tribe/Main.php

```diff
@@ -508,6 +504,17 @@ protected function init_autoloading() {
 			$autoloader->register_autoloader();
 		}
 
+		/**
+		 * Load The Events Calendar text domain.
+		 *
+		 * @since TBD
+		 *
+		 * @return bool
+		 */
+		public function load_text_domain() {
+			return Tribe__Main::instance( $this )->load_text_domain( 'the-events-calendar', $this->plugin_dir . 'lang/' );
```

Comment:
Warning here -- recursion issue!

`Tribe__Main::load_text_domain()` calls the `tribe_load_text_domain` filter